### PR TITLE
Connect AppVeyor to Pebble project

### LINKED ISSUE #208: Design of Windows CI/CD pipeline for Pebble
The general idea of this design proposition, is to setup an AppVeyor CI/CD pipeline that ultimately publish on Docker Hub a Windows Container for Pebble, and another one for Pebble-Challtestsrv.

Indeed my recent tests to build Certbot on Windows showed that Pebble can run on Windows out-of-the-box, and so is a suitable integration tests ACME CA server for this platform. The preferred way to distribute the software would be Windows Containers available on Docker Hub side by side to their Linux counterparts.

I see two general approaches for the AppVeyor CI:
1) Create an equivalent pipeline to Travis CI for Linux
2) Focus only on the Docker Hub publication

For 1), it means to translate each step of the Travis CI/CD pipeline to the AppVeyor CI/CD one:
* setup hosts and environment variables
* clone Certbot, setup its venv, enable it
* go install pebble and start the generated binary
* run chisel2.py
* build from source the docker, and publish it

The way AppVeyor is configured is really close to the Travis approach, consuming a `appveyor.yml` YAML file with typical steps of run, build, test and deploy. So the technology allows to follow the same general approach. In the details, apart from Go which is built to be cross-platform out-of-the-box:
* Certbot is already working on Windows, on a satisfactory test-grade level (almost all unit tests are passing, in `acme` module is already fully compatible)
* from what I see, `chisel2.py` has no platform specific code
* Docker is available and running on standard AppVeyor Windows platform

On the cons, the full build logic needs to be maintained on both AppVeyor and Travis. On the pros, Pebble is fully tested on Windows before its publication to Docker Hub.

For 2), the idea would be to first poll the relevant build status from Travis, using its API (see https://docs.travis-ci.com/user/developer/#api-v3). When the final status (succeed/failed) is obtained, AppVeyor proceed this way:
* in case of Travis build failure, the AppVeyor build is failed
* in case of Travis build succeed, AppVeyor build and publish the Windows Docker, in a really similar way to the last step of the solution 1).

On the cons, Pebble is not tested on Windows, only relying on the results obtained on Linux. On the pros, AppVeyor config contains less logic, and does not need to be maintained in parallel to the Travis on. Also, the tests are relying on Certbot for Linux, which is at a production-grade level.

I prefer personnally 1) for the Windows testability, but I wanted to propose an alternative with less AppVeyor knowledge commitment.

As last words, you can see on by fork a running example of AppVeyor building and pushing Pebble + Pebble-Challtestsrv Windows Docker upon GIT push, and operable relevant Dockerfiles for Windows.

@jsha, @cpu, I am open and available for discussions!

### PR DESCRIPTION
**Pull request by adferrand**
This PR is the prerequisite to #208, it is about activating a Windows pipeline for Pebble. This PR contains only a dummy pipeline that is here to confirm that everything is wired. The actual logic for this pipeline will be in the next PR.

AppVeyor is a CI provider that uses Windows VMs to execute specific pipelines for this platform. We use it in the Certbot project. Everything is configured through an `appveyor.yml` file, similar to `.travis.yml`.

I put here a link to the equivalent PR that I did for Certbot when we connected AppVeyor for it: certbot/certbot#6361. It contains the detailed instructions for the steps I am about to enumerate here:

Basically, activating AppVeyor consists in:
* creating a `letsencrypt` account on https://www.appveyor.com/,
* associating the `letsencrypt` GitHub account using OAuth,
* create an AppVeyor project for the `letsencrypt/pebble` GitHub project
* merging this PR

Once done, I will check that the dummy pipeline is correctly triggered for the `master` branch. Then I will update my PR against `master` to trigger the first full pipeline logic for a PR push.

### CODE REVIEW

**Comment by cpu** [comment]
I think next step is :mag:ing and merging https://github.com/letsencrypt/pebble/pull/213

**Comment by adferrand** [comment]
@jsha, @cpu I check on #213 for correct execution of the AppVeyor pipeline.
